README.md

```diff
@@ -1,2 +1,62 @@
 # Hello_Microverse
 Hello World
+
+
+
+
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo 
+
+[Coming Soon]
+
+
+## Getting Started
+
+**Go to your Cli and follow the steps below**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repository: go to Your Cli and run "git clone <repository url>"
+
+- change into the repository:  run 'cd <repository name>'
+
+- Install all the project dependences: Run 'npm Insttall' in your cli
+
+- now you can view the project in the browser using any web server of choice
+
+
+
+## Author
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/Eugeneiyukhun)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/eugene-iyukhun-bb491612a)
+- Twitter: [twitter](https://twitter.com/EIyukhun)
```

Comment:
[OPTIONAL] Providing actual usernames for your social media handles instead of `@githubhandle` and `twitter` etc. makes it easy to navigate and more proffessional :+1: Kindly consider updating this section accordingly